packages/components/src/components/text/text.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React, { HTMLProps } from 'react';
+import { isEmptyObject } from '@deriv/shared';
+import { TGenericStringValueObject } from '../types';
+
+interface ITextProps extends HTMLProps<HTMLElement> {
```

Comment:
let's stick to `type` instead of `interface`
```suggestion
type TTextProps extends HTMLProps<HTMLElement> {
```

Reply:
Yeah @matin-binary. This was resolved.